sdcm/utils/common.py

```diff
@@ -1232,6 +1232,29 @@ def get_s3_scylla_repos_mapping(dist_type='centos', dist_version=None):
     return _S3_SCYLLA_REPOS_CACHE[(dist_type, dist_version)]
 
 
+ScyllaProduct = Literal['scylla', 'scylla-enterprise']
+
+
+def get_latest_scylla_release(region: str = 'eu-west-1',
+                              product: ScyllaProduct = 'scylla') -> str:
+    """
+        Get the latest release, base on the formal AMIs published
+    """
+    if product == 'scylla-enterprise':
+        filter_regex = re.compile(r"(rc|dev)", flags=re.IGNORECASE)
+        version_regex = re.compile(r"enterprise\s*(\d*\.\d*\.\d*)", flags=re.IGNORECASE)
+    else:
+        filter_regex = re.compile(r"(enterprise|rc|dev)", flags=re.IGNORECASE)
+        version_regex = re.compile(r"(\d*\.\d*\.\d*)", flags=re.IGNORECASE)
+    versions = []
+    for ami in get_scylla_ami_versions(region_name=region):
+        if not filter_regex.search(ami.name.lower()):
+            if version := version_regex.search(ami.name.lower()):
+                versions.append(Version(version.group(1)))
+    versions.sort()
+    return str(versions[-1])
```

Comment:
Um, do you want to return biggest one? Will `return max(versions)` work for you?

Reply:
probably yes, it should work the same.

Reply:
fixed